src/chrome/content/rules/Visa.com.xml

```diff
@@ -0,0 +1,55 @@
+<!--
+	Other Visa rulesets:
+		- Visa_Buxx.com.xml
+		- Visa_Europe.xml
+
+	403:
+		developer.visa.com
```

Comment:
Maybe you can use a static resource as test url, like `https://developer.visa.com/assets/css/portal.css`